mvc/system/classes/rout.php

```diff
@@ -1,9 +1,9 @@
 <?php
 class rout
 {
-  public $controller = "welcome";
+  public $controller = "home";
```

Comment:
We can use protected as well since we don't need to access these variables outside the class.